libs/designer/src/lib/core/actions/bjsworkflow/serializer.ts

```diff
@@ -306,7 +306,13 @@ const serializeManifestBasedOperation = async (rootState: RootState, operationId
   const nodeStaticResults = getRecordEntry(rootState.operations.staticResults, operationId) ?? ({} as NodeStaticResults);
   const inputPathValue = serializeParametersFromManifest(inputsToSerialize, manifest);
   const hostInfo = serializeHost(operationId, manifest, rootState);
-  const inputs = hostInfo !== undefined ? mergeHostWithInputs(hostInfo, inputPathValue) : inputPathValue;
+  let inputs = hostInfo !== undefined ? mergeHostWithInputs(hostInfo, inputPathValue) : inputPathValue;
+  const retryPolicy = getRetryPolicy(nodeSettings);
+  if (retryPolicy) {
+    inputs.retryPolicy = retryPolicy;
+  }
+  inputs = swapInputsLocationIfNeeded(inputs, manifest.properties.inputsLocationSwapMap);
```

Comment:
I didnt understand previously also retry policy was getting removed before swapINputs, could you explain what else needs to be happen now after swapInputs?